metadata/config.yaml

```diff
@@ -5,7 +5,7 @@ CONFIG_NAME: hoodi
 # Genesis
 # ---------------------------------------------------------------
 # `2**14` (= 16,384)
-MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 1000000
+MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 16384
```

Comment:
why are we changing this now? we already cut a rc yesterday which people might be running with the current value

Reply:
Just following this:
https://github.com/eth-clients/hoodi/pull/5#issuecomment-2728548465

Reply:
Ok maybe it does not matter, looks like the only thing we do in Lodestar with `MIN_GENESIS_ACTIVE_VALIDATOR_COUNT` is "waiting for genesis validators" but since both values `16384` and `1000000` are fulfilled it should be fine to run with previous config

Reply:
thank you! approving and merging the PR then